.github/workflows/continuous-integration-workflow.yml

```diff
@@ -60,6 +60,53 @@ jobs:
           retention-days: 1
 
 
+  integration:
+    name: Execute Integration CI tests
+    runs-on: ubuntu-latest
+    needs: build
+    permissions:
+      contents: read
+      packages: read
+
+    steps:
+      # The first 2 steps (checking out, and building from Openfire Dockerfile)
+      # can be removed when the docker image is published to a registry.
+      # This 'integration' job should then need the 'build-and-push-docker' job as a dependency.
+      - uses: actions/checkout@v4
+
+      - name: Build Docker Image
+        run: |
+          docker build -t openfire:latest .
+
+      - name: Checkout Integration Tests
+        uses: actions/checkout@v4
+        with:
+          repository: surevine/openfire-integration-tests
```

Comment:
Just spotted that this needs changing now that we have migrated the repository to IR. I'll push a change.